### PR TITLE
Limit sharing institutes to institute settings choices

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR connects institute settings chosen sharing institutes to the institute sharing options on the case page.

Created on the fly on the train and not tested.. Might or might not work!

**How to test**:
Test on stage and try it

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by
- [x] tests executed by

### CODE REVIEW

File: scout/server/blueprints/cases/controllers.py [hunk 1]
```diff
@@ -189,8 +189,7 @@ def case(store, institute_obj, case_obj):
                                    collab_obj in o_collaborators]
 
     collab_ids = [(collab['_id'], collab['display_name']) for collab in store.institutes() if
-                  (collab['_id'] not in ('cust000', institute_obj['_id'])) and
-                  (collab['_id'] not in case_obj['collaborators'])]
+        collab['_id'] in institute_obj.get('collaborators')
```
Comment (dnil) [inline scout/server/blueprints/cases/controllers.py]: :+1:

Comment (dnil) [reply scout/server/blueprints/cases/controllers.py]: You are fast - but you shouldn't really be working on this.. I just found the right function a minute ago, but was checking what o_collaborators did. Turns out just for the revoke I, so this is perfect!

Comment (northwestwitch) [reply scout/server/blueprints/cases/controllers.py]: Last contributions, and probanly doesn't work, shoulf be fixed

Comment (dnil) [reply scout/server/blueprints/cases/controllers.py]: Ah, right, tried it now. But, right place. I'm on it!

**Comment by dnil** [comment]
It now works locally!

Screenshot 2020-01-31 at 18 19 51

**Comment by dnil** [comment]
Looks good at stage as well!
Screenshot 2020-01-31 at 18 34 58

**Comment by dnil** [comment]
Merging in 3..2..

**Pull request merged**